src/components/Tokens/state.ts

```diff
@@ -4,6 +4,7 @@ import { atomWithReset } from 'jotai/utils'
 import { useCallback } from 'react'
 
 export enum TokenSortMethod {
```

Comment:
more of tech debt and only exposed by this PR, but `TokenSortMethod` should not be the key for `HEADER_DESCRIPTIONS`, there's very little overlap

Reply:
should it not be? HEADER_DESCRIPTIONS represents the hover-over descriptions for the variables in the token table header, which is what TokenSortMethod represents